azurelinuxagent/common/exception.py

```diff
@@ -55,7 +55,11 @@ class CGroupsException(AgentError):
 
     def __init__(self, msg, inner=None):
         super(AgentError, self).__init__(msg, inner)
+        # TODO: AgentError should set the message - investigate whether doing it there would break anything
```

Comment:
So... would it? :)

Reply:
Dunno; no time to check :)

Doing it  in AgentError would change the formatting of all exceptions. Need to spend some time checking what would happen then.